Workbooks/Virtual Machines/Availability/Server Availability.workbook

```diff
@@ -274,5 +265,8 @@
       "name": "query - 0"
     }
   ],
+  "fallbackResourceIds": [

```

Comment:
you don't need this `fallbackResourceIds` section either, surprising it didn't warn in the validation build.  no harm here specifically but surprising nontheless.

Reply:
seems to be there every time I copy from the portal, i'll remove it